src/iconsManifest/supportedExtensions.ts

```diff
@@ -1360,7 +1360,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'drawio',
-      extensions: ['drawio', 'dio'],
+      extensions: ['drawio', 'drawio.svg', 'drawio.png', 'dio', 'dio.svg', 'dio.png'],
       filenamesGlob: ['.drawio'],
       extensionsGlob: [],
```

Comment:
Simplify this for the same result!
```suggestion
      extensions: ['drawio', 'dio'],
      filenamesGlob: ['.drawio', '.dio'],
      extensionsGlob: ['png', 'svg'],
```

Reply:
Test locally that the above satisfies `.drawio`, `.drawio.svg`, `.drawio.png`